config.py

```diff
@@ -26,9 +30,13 @@ class DevelopmentConfig(Config):
     LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'DEBUG')
     BACKSTAGE_API_URL = os.getenv('BACKSTAGE_API_URL', 'http://localhost:8001/backstage-api')
     SECURE_MESSAGE_URL = os.getenv('SECURE_MESSAGE_URL', 'http://localhost:5050')
+    COLLECTION_EXERCISE_SERVICE = os.getenv('COLLECTION_EXERCISE_SERVICE', 'http://localhost:8145')
```

Comment:
Should we keep the env var naming convention consistent with ones that are already there? COLLECTION_EXERCISE_URL?